gradio/analytics.py

```diff
@@ -27,15 +27,14 @@ def analytics_enabled() -> bool:
 
 
 def _do_analytics_request(url: str, data: dict[str, Any]) -> None:
+    data["ip_address"] = get_local_ip_address()
```

Comment:
Makes sense so that launching the thread does not have to wait for this to complete 👍